src/icons/ChevronDown.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ChevronDown = ({ height, ...other }) => (
+  <svg
+    height={height}
```

Comment:
Any reason for height here?

Reply:
Added height and width default to be 24px (1.5rem), the natural SVG size for these specific SVG icons